### ISSUE TITLE
colors in python language

### ISSUE THREAD

**Issue by YuriFontella** [franzgollhammer/jb-fleet-dark#3]
Your color theme was very good, but the intention really wasn't to be as faithful to the original colors?
Just changing the lilac/purple to yellow in the functions would make the theme perfect.

![image](https://github.com/franzgollhammer/jb-fleet-dark/assets/4920619/9a430fb7-caba-44ca-83a4-210acf3b6ea1)

thanks.

**Comment by vioo-bkp** [comment]
indeed. Also for not creating a new thread, I suggest reducing the number of themes (13 themes are too many imo).
I suggest four themes: dark, light, neutral (with a bit of contrast), and Monokai derivation.

**Comment by franzgollhammer** [comment]
Thanks for the suggestion! I actually reduced the number of themes for maintainability reasons, so I fully understand your point. JetBrains Fleet itself is no longer actively developed, but I’ll continue keeping the theme maintained as best as possible. Thanks! 

KR
F
